modules/osdk-updating-v1101-to-v1160.adoc

```diff
@@ -0,0 +1,195 @@
+    annotations:
+      kubectl.kubernetes.io/default-container: manager
+...
+----
+
+. Add `PHONY` targets to all of the targets in your `Makefile` file.
+
+. For Go-based Operator projects, make the following changes:
+
+.. Install the `setup-envtest` binary.
+
+.. Change your `go.mod` file to update the dependencies:
++
+[source,golang]
+----
+k8s.io/api v0.22.1
+k8s.io/apimachinery v0.22.1
+k8s.io/client-go v0.22.1
+sigs.k8s.io/controller-runtime v0.10.0
+----
+
+.. Run the `go mod tidy` command to download the dependencies:
++
+[source,terminal]
+----
+$ go mod tidy
+----
+
+.. Make the following changes to your `Makefile` file:
++
+[source,diff]
+----
+...
+- ENVTEST_K8S_VERSION = 1.21`
++ ENVTEST_K8S_VERSION = 1.22`
+
+  test: manifests generate fmt vet envtest ## Run tests.
+-   go test ./... -coverprofile cover.out
++   KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+...
+
+- $(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
++ $(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+...
+
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+- CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+...
+- admissionReviewVersions={v1,v1beta1}
++ admissionReviewVersions=v1
+...
+
++ ifndef ignore-not-found
++   ignore-not-found = false
++ endif
+
+##@ Deployment
+...
+- sh kubectl delete -f -
++ sh kubectl delete --ignore-not-found=$(ignore-not-found) -f -
+----
+
+.. Run the `make manifest` command to generate your manifests with the updated version of Kubernetes:
++
+[source,terminal]
+----
+$ make manifest
+----
+
+. For Ansible-based Operator projects, make the following changes:
++
+.. Change your `requirements.yml` file to include the following:
+
+... Replace the `community.kubernetes` collection with the `kubernetes.core` collection:
++
+[source,yaml]
+----
+...
+- name: kubernetes.core
+  version: "2.2.0"
+...
+----
+
+... Update the `operator_sdk.util` utility from version `0.2.0` to `0.3.1`:
++
+[source,yaml]
+----
+...
+- name: operator_sdk.util
+  version: "0.3.1"
+----
+
+.. Verify the default resource limits in the `config/manager/manager.yaml` file:
++
+[source,yaml]
+----
+...
+ # TODO(user): Configure the resources accordingly based on the project requirements.
+ # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 768Mi
+  requests:
+    cpu: 10m
+    memory: 256Mi
+----
++
+[IMPORTANT]
+====
+Operator SDK scaffolds these values as a reasonable default setting. Operator authors should set and optimize resource limits based on the requirements of their project.
+====
+
+.. Optional: Make the following changes if you want to run your Operator locally by using the `make run` command:
```

Comment:
Should this Optional step at the same level as the step above it: For Ansible-based Operator projects, make the following changes?